From 7e5fbdd1bb67c2a53101548063f130d7a4d9b14f Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Wed, 10 Sep 2014 00:17:42 +0100
Subject: [PATCH 1/2] autotools: Remove broken 'unchanged' check

Before m_buildDirectory gets changed buildDirectory() returns
a default of the projectDir and notifyBuildDirectoryChanged()
would not get called until the build directory was editted.

Change-Id: Ie31fb2722c663cd0f998458d536181f47c83a37d
---
 src/plugins/autotoolsprojectmanager/autotoolsbuildconfiguration.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/plugins/autotoolsprojectmanager/autotoolsbuildconfiguration.cpp b/src/plugins/autotoolsprojectmanager/autotoolsbuildconfiguration.cpp
index aa593c1..dca4766 100644
--- a/src/plugins/autotoolsprojectmanager/autotoolsbuildconfiguration.cpp
+++ b/src/plugins/autotoolsprojectmanager/autotoolsbuildconfiguration.cpp
@@ -226,8 +226,6 @@ BuildConfiguration::BuildType AutotoolsBuildConfiguration::buildType() const
 
 void AutotoolsBuildConfiguration::setBuildDirectory(const Utils::FileName &directory)
 {
-    if (directory == buildDirectory())
-        return;
     BuildConfiguration::setBuildDirectory(directory);
     ProjectExplorer::BuildStepList *bsl = stepList(ProjectExplorer::Constants::BUILDSTEPS_BUILD);
     foreach (BuildStep *bs, bsl->steps()) {
-- 
2.1.0

